Testscripts/Linux/utils.sh

```diff
@@ -3391,3 +3391,62 @@ function Format_Mount_NVME()
     return 0
 }
 
+
+# Remove and reattach PCI device inside the VM
+# @param1 DeviceType: supported "NVME", "SR-IOV"
+#                    ("GPU" may be supported in the future)
+# @return 0 the device was removed and reattached successfully
+function RescindPCI ()
+{
+    if [ "$1" = "SR-IOV" ]; then
```

Comment:
use a case switch statement, is easier to read and more well suited for string params